modules/components/src/Aggs/BooleanAgg.css

```diff
@@ -0,0 +1,38 @@
+.booleanFacetWrapper {
+  display: flex;
+  flex-direction: row;
+  height: 30px;
+  font-size: 0.9em;
+}
+.booleanFacetAny {
```

Comment:
a lot of shared styles between these. Could be good to make one selector for the general stuff